### PR TITLE
hotfix-flip-card-malfunction

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: src/STCard.ts [hunk 1]
```diff
@@ -63,7 +63,6 @@ class STCard {
    */
   public onSecurityCodeChange(value: string, outsideValue: boolean) {
     this._card.onSecurityCodeChanged(value, outsideValue);
-    this._card.flipCard();
```
Comment (pgs-gmartowski) [inline src/STCard.ts]: info: this was duplicated and in some cases card flips twice :/

File: src/models/Card/Card.ts [hunk 2]
```diff
@@ -18,7 +18,11 @@ class Card extends Utils {
   private static DEFAULT_LANGUAGE: string = 'en_GB';
   private static DISABLED_ATTRIBUTE: string = 'disabled';
   private static ERROR_CLASS: string = 'error';
-  private static NOT_FLIPPED_CARDS: string[] = [CARD_TYPES.AMEX];
+  private static NOT_FLIPPED_CARDS: string[] = [CARD_TYPES.AMEX, CARD_TYPES.PIBA];
```
Comment (pgs-gmartowski) [inline src/models/Card/Card.ts]: info: there is no need to flip card if PremierInn card is indicated

File: src/models/Card/Card.ts [hunk 3]
```diff
@@ -18,7 +18,11 @@ class Card extends Utils {
...
+
+  private static _clearInputValue(id: string) {
+    (document.getElementById(id) as HTMLInputElement).value = '';
```
Comment (pgs-gmartowski) [inline src/models/Card/Card.ts]: info: after indicating PIBA card, CVV field was not cleared

File: src/styles/card.scss [hunk 4]
```diff
@@ -14,7 +14,6 @@ body {
   border: 0;
   overflow: hidden;
   min-height: 30rem;
-  min-width: 37rem;
```
Comment (pgs-gmartowski) [inline src/styles/card.scss]: info: minor fix in CSS - Marek noticed that card is not looking good on iPhones

File: src/models/Card/Card.ts (hunk 2 above)
Comment (james-simpson-ST) [reply src/models/Card/Card.ts]: Agreed :) Nice

**Pull request merged**